e2e-tests/tests/test_jolteon_advanced.py

```diff
@@ -0,0 +1,295 @@
+from time import sleep, time
+from pytest import mark
+from src.blockchain_api import BlockchainApi, Wallet
+from config.api_config import ApiConfig
+import logging as logger
+
+
+@mark.jolteon
```

Comment:
Do not forget to add all custom tags into pytest.ini and in the docs (describing what it for)